build/build-modules-js/stylesheets/scss-transform.es6.js

```diff
@@ -10,7 +10,7 @@ module.exports.compile = async (file) => {
 
   let compiled;
   try {
-    compiled = Sass.renderSync({ file });
+    compiled = Sass.compile({ file });
```

Comment:
```suggestion
    compiled = Sass.compile(file);
```
Shouldn’t it be like with the other 2 changes